src/theme/SearchBar/style.css

```diff
@@ -72,15 +96,22 @@
   color: var(--ifm-font-color-base);
   font-family: var(--ifm-heading-font-family);
   font-weight: var(--ifm-heading-font-weight);
-  margin-top: 0.5em;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+}
+
+.meilisearch-autocomplete .docs-searchbar-suggestion--subcategory-column {
+  width: 30%;
+  text-align: right;
+  position: relative;
+  padding: 5.3333333333px 10.6666666667px;
```

Comment:
That's some very precise padding

Reply:
yeah, i just copy pasted it hoping it works in our custom-ish css too 😂